packages/core/admin/admin/src/content-manager/components/EditViewDataManagerProvider/index.js

```diff
@@ -290,23 +290,38 @@ const EditViewDataManagerProvider = ({
       e.preventDefault();
       let errors = {};
 
-      // First validate the form
       try {
         await yupSchema.validate(modifiedData, { abortEarly: false });
+      } catch (err) {
```

Comment:
We'd need to separate this validation from `onPost` and `onPut` to be able to throw multiple errors.

Reply:
Yes but we need to avoid to send the data if the yup validation fails unless we remove this completely.

Reply:
Fixed that in 6f02f61af7bd56e135e2a79c0c568078187f0a21